python/src/multi_agent_orchestrator/orchestrator.py

```diff
@@ -137,10 +137,13 @@ async def agent_process_request(self,
                                user_id: str,
                                session_id: str,
                                classifier_result: ClassifierResult,
-                               additional_params: dict[str, str] = {},
+                               additional_params: dict[str, str] | None = None,
```

Comment:
Should not add mutable parameters in the method signature. Moving the check to inside the function.